unitTests/Test.ts

```diff
@@ -6,6 +6,9 @@ if (Simulate.isChromeHeadless()) {
   Logger.disable();
 }
 
+import { MockCookie } from './MockCookie';
+MockCookie.mockBrowserCookies();
```

Comment:
I am calling the mock here rather than in an `it`/`beforeEach`/etc block, since for some reason it's impossible to mock cookies in those contexts. `CookieUtilsTest` also mocked them outside of a `it`/`beforeEach`/etc block, but mocked them at the beginning of its own file. I am mocking them in `Test.ts` to have consistency between every test.